content/en/docs/concepts/services-networking/service-topology.md

```diff
@@ -15,7 +15,15 @@ weight: 10
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.17" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.21" state="deprecated" >}}
+
+{{< note >}}
+
+Service Topology is deprecated as of Kubernetes v1.21, and will be removed in
```

Comment:
Relatively minor: I'd prefer we not say that "Service Topology is deprecated" but instead something like "The alpha topologyKeys API is deprecated"

Reply:
Updated, thanks!